README.md

```diff
@@ -254,6 +254,30 @@ tensorboard in inspect mode to inspect the contents of your event files.
 
 `tensorboard --inspect --logdir DIRECTORY_PATH`
 
+An empty event file (from a Keras job that starts output at batch 2) might look like this:
```

Comment:
It might be worth a little clarification here that this event file isn't exactly "empty" (in that it does contain some events, as indicated by the non-zero steps) but that it doesn't show any summary data.

E.g. perhaps word as something like:

> The output for an event file corresponding to a blank TensorBoard may still sometimes show a few steps, representing a few initial events that aren't shown by TensorBoard (for example, when using the Keras TensorBoard callback):
>
> [output]
>
> In contrast, the output for an event file with more data might look like this:
>
> [output]

Reply:
Thanks.  Much better now.